bench/tiered/push_pull.c

```diff
@@ -214,11 +215,11 @@ recover_validate(const char *home, uint32_t num_records, uint64_t file_size, int
     testutil_wiredtiger_open(opts, home, buf, NULL, &conn, true, true);
     testutil_check(conn->open_session(conn, NULL, NULL, &session));
 
-    // srand((unsigned long)getpid() + num_records);
-    srand((uint32_t)getpid() + num_records);
+    /* Seed the random number generator */
+    rnd.v = (uint32_t)getpid() + num_records;
     str_len = sizeof(data_str) / sizeof(data_str[0]);
     for (i = 0; i < str_len - 1; i++)
-        data_str[i] = 'a' + (uint32_t)rand() % 26;
+        data_str[i] = 'a' + (uint32_t)__wt_random(&rnd) % 26;
```

Comment:
So given my previous comment, one might think we shouldn't use random content at all, maybe just push a bunch of zeros or "AAAAA" into the string.  But then WT (or AWS) could "cheat" - they are free to use any compression tactics they like to give better service.  So we _should_ have random data.   However, letters 'a' through 'z' only provide some randomness, and we're using the same string every time.  I'm going to write a comment in the ticket, I think we need to do something a bit different to get our random data.